include/api.php

```diff
@@ -3845,7 +3846,7 @@ function api_friendica_notification_seen($type){
 	 */
 	function api_friendica_direct_messages_setseen($type){
 		$a = get_app();
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.